src/main/kotlin/com/terraformation/backend/api/Annotations.kt

```diff
@@ -74,6 +74,14 @@ annotation class ApiResponse400(
     val description: String = "The request was not valid."
 )
 
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.FUNCTION)
+@ApiResponseSimpleError(responseCode = "403")
+annotation class ApiResponse403(
```

Comment:
Good call. We've been bad about including this in the schema in the API in general.